docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,315 @@
-Neptune-PyTorch Lightning Integration
-=====================================
+PyTorch Lightning Integration
+=============================
 
-The integration enables you to log |PyTorch Lightning| metrics to Neptune.
-
-.. image:: ../_static/images/integrations/pytorchlightning_neptuneml.png
-   :target: ../_static/images/integrations/pytorchlightning_neptuneml.png
+What will you get?
+------------------
+.. image:: ../_static/images/integrations/lightning_basic.gif
```

Comment:
is this .gif available? I don't see it in Files changed for this PR

Reply:
Also, perhaps it would be better to just create a YT and embed -> I think this is a longer "piece" and having a .gif here is not the best option (as we discussed before)

Reply:
ups, sorry did not put them here. For now it's gif, I'll do video at the end.

Reply:
Then YT and embed of course.